spec/integration/precompilation_spec.rb

```diff
@@ -43,4 +43,26 @@
     sprite_files = Dir.glob(File.join('public', 'assets', 'sprites', 'application-*.png'))
     expect(sprite_files.length).to eq(2)
   end
+
+  context 'a file is renamed' do
```

Comment:
The reason I have to move the file twice is that the renaming problem doesn't occur until then. I've updated the context to indicate that.